EduNLP/I2V/i2v.py

```diff
@@ -115,23 +134,59 @@ def from_pretrained(cls, name, model_dir=MODEL_DIR, *args, **kwargs):
     "d2v_eng_256": [D2V, "d2v_eng_256"],
     "d2v_lit_256": [D2V, "d2v_lit_256"],
     "w2v_sci_300": [W2V, "w2v_sci_300"],
-    "w2v_lit_300": [W2V, "w2v_lit_300"],
+    "w2v_lit_300": [W2V, "w2v_lit_300"]
 }
 
 
-def get_pretrained_i2v(name, model_dir=MODEL_DIR):
+MODEL_TYPES = {
+    "d2v": D2V,
+    "w2v": W2V
+}
+
+
+def get_pretrained_i2v(name=None, model_dir=MODEL_DIR, source=None,
```

Comment:
Considering the use of overloading is not so suitable beacuse the params is the same in number and type when use models from remote or local,  so I maintain these params in one function like before. And user can learn the detailed usage from comments, api and examples.

Reply:
This part is not elegent enough, revert it and I will handle this.